app/views/shared/_flash.html.erb

```diff
@@ -7,7 +7,7 @@
       else "usa-alert--info"
       end %>
 
-    <div class="grid-container usa-alert <%= alert_class %> usa-alert--slim">
+    <div class="grid-container usa-alert <%= alert_class %> usa-alert--slim maxw-tablet">
```

Comment:
![Screenshot 2025-03-10 at 7 46 56 PM](https://github.com/user-attachments/assets/b0e1dda9-abd3-4344-b673-c90fb57691cd)
let's revert this one

Reply:
actually for the score revision success message it should show this alert instead of the share flash message:
![Screenshot 2025-03-10 at 7 50 32 PM](https://github.com/user-attachments/assets/f469f034-bf57-4e82-9b97-7e552a64525e)